projects/schematics/src/shared/utils/config-utils.ts

```diff
@@ -336,3 +337,28 @@ export function getSpartacusProviders(sourceFile: SourceFile): Node[] {
   }
   return providers;
 }
+
+const EMPTY_SPACE_REG_EXP = /\s+/gm;
+export function normalizeConfiguration(config: string | Node): string {
+  let newConfig: string;
+  if (typeof config === 'string') {
+    newConfig = config;
+  } else {
+    if (Node.isTypeAssertion(config) || Node.isAsExpression(config)) {
+      newConfig = config.getExpression().getText();
+    } else {
```

Comment:
Is it used?

Reply:
Yes, for the cases when the config doesn't have the `as syntax` nor the `type assertion`.